src/app.js

```diff
@@ -1,299 +1,364 @@
+        collectionId,
+        'name',
+        255,
+        false,
+        "Empty Name",
+        false
+    );
+    console.log(nameAttributeResponse);
+
+    const yearAttributeResponse = await databases.createIntegerAttribute(
+        databaseId,
+        collectionId,
+        'release_year',
+        false,
+        0, 5000,
+        1970,
+        false
+    );
+    console.log(yearAttributeResponse);
+
+    console.log("Waiting a little to ensure attributes are created ...");
+    await new Promise((resolve) => setTimeout(resolve, 2000));
 }
 
 const listCollections = async () => {
-  console.log(chalk.greenBright('Running List Collections API'));
+    console.log(chalk.greenBright('Running List Collections API'));
+
+    const response = await databases.listCollections(databaseId);
 
-  const database = new Databases(client, "default");
-  const response = await database.listCollections();
-  console.log(response);
+    console.log(response);
 }
 
 const listAttributes = async () => {
-  console.log(chalk.greenBright('Running List Attributes API'));
+    console.log(chalk.greenBright('Running List Attributes API'));
+
+    const response = await databases.listAttributes(databaseId, collectionId);
 
-  const database = new Databases(client, "default");
-  const response = await database.listAttributes(collectionId);
-  console.log(response);
+    console.log(response);
 }
 
 const getAccount = async () => {
-  console.log(chalk.greenBright('Running Get Account API'));
+    console.log(chalk.greenBright('Running List Users API'));
 
-  const account = new Account(client);
-  const response = await account.get();
-  console.log(response);
+    const response = await account.get();
+
+    console.log(response);
 }
 
 const addDocument = async () => {
-  console.log(chalk.greenBright('Running Add Document API'));
-
-  const database = new Databases(client, "default");
-  const response = await database.createDocument(collectionId,
-    "unique()",
-    {
-      name: 'Spider Man',
-      release_year: 1920
-    },
-    ['role:all'], ['role:all']
-  );
-  console.log(response);
-  documentId = response.$id;
+    console.log(chalk.greenBright('Running Add Document API'));
+
+    const response = await databases.createDocument(
+        databaseId,
+        collectionId,
+        ID.unique(),
+        {
+            name: 'Spider Man',
+            release_year: 1920
+        },
+        [
+            Permission.read(Role.any()),
+            Permission.update(Role.users()),
+            Permission.delete(Role.users()),
+        ]
+    );
+    documentId = response.$id;
+
+    console.log(response);
 }
 
 const listDocuments = async () => {
-  console.log(chalk.greenBright('Running List Documents API'));
+    console.log(chalk.greenBright('Running List Documents API'));
+
+    const response = await databases.listDocuments(
+        databaseId,
+        collectionId,
+        [
+            Query.equal('release_year', 1920),
```

Comment:
When I ran this, I got an error saying the index on `release_year` was not found:

```
AppwriteException [Error]: Index not found: release_year
    at Client.call (/workspace/playground-for-node/node_modules/node-appwrite/lib/client.js:171:31)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Databases.listDocuments (/workspace/playground-for-node/node_modules/node-appwrite/lib/services/databases.js:1004:16)
    at async listDocuments (/workspace/playground-for-node/src/app.js:128:22)
    at async runAllTasks (/workspace/playground-for-node/src/app.js:331:5) {
  code: 400,
  type: 'general_argument_invalid',
  response: {
    message: 'Index not found: release_year',
    code: 400,
    type: 'general_argument_invalid',
    version: '1.0.1'
  }
}
```